sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -64,17 +72,26 @@ public String getPlatform() {
         return platform;
     }
 
+    public Map<String, Object> getLocals() {
+        return locals;
+    }
+
     /**
      * Convert an array of {@link StackTraceElement}s to {@link SentryStackTraceElement}s.
      *
      * @param stackTraceElements Array of {@link StackTraceElement}s to convert.
+     * @param cachedFrames Array of cached {@link Frame}s (from the Sentry agent) if available,
+     *                     or null.
```

Comment:
I think it'd helpful to add a constructor with a single parameter to make this a little more explicit but I'm not going to block on it.